lib/ex_unit/test/ex_unit/doc_test_test.exs

```diff
@@ -548,6 +548,26 @@ defmodule ExUnit.DocTestTest.PatternMatching do
 end
 |> ExUnit.BeamHelpers.write_beam()
 
+defmodule ExUnit.DocTestTest.Ellipsis do
+  @doc """
+      iex> ExUnit.DocTestTest.Ellipsis.same_line_err(self())
+      ** (ArgumentError) Unexpected: ...
+  """
+  def same_line_err(arg) do
+    raise ArgumentError, message: "Unexpected: #{inspect(arg)}"
+  end
+
+  @doc """
+      iex> ExUnit.DocTestTest.Ellipsis.multi_line_err(self())
+      ** (ArgumentError) Unexpected:
+      ...
+  """
+  def multi_line_err(arg) do
+    raise ArgumentError, message: "Unexpected:\n#{inspect(arg)}"
+  end
+end
+|> ExUnit.BeamHelpers.write_beam()
```

Comment:
Should we add a test when the error message itself has "..." at the end?

Reply:
Added in [468bfbc](https://github.com/elixir-lang/elixir/pull/14233/commits/468bfbc14fb8eb4db509cfcb65959f5e0ffcd85f).
I'm not sure it's too useful since technically if we allow for anything then there's little reason for `...` not to pass, but it doesn't cost much to have it so I'm fine either way.